aliases/ecs-deploy.php

```diff
@@ -0,0 +1,41 @@
+<?php
+date_default_timezone_set('America/New_York');
+
+require_once __DIR__ . '/ecs-db-backup.php';
+
+$AwsGateway = require_once __DIR__ . '/../services/awsGateway.php';
+
+$aws = new AwsGateway();
+$project = $argv[1];
+$environment = $argv[2];
+$version = $argv[3];
+$serviceName = "{$project}-{$environment}";
+
+$service = $aws->findService($serviceName);
```

Comment:
it would probably simplify your life to have one of these that actually accepts project and environment as args